cmd/guaccollect/cmd/files.go

```diff
@@ -81,7 +85,7 @@ you have access to read and write to the respective blob store.`,
 		logger := logging.FromContext(ctx)
 
 		// Register collector
-		fileCollector := file.NewFileCollector(ctx, opts.path, opts.poll, 30*time.Second)
+		fileCollector := file.NewFileCollector(ctx, opts.path, opts.poll, 30*time.Second, opts.useBlobURL)
```

Comment:
this will need to happen with the other collectors also. Either being called from `guaccollect` or `guacone` (just to be safe).

Reply:
I am not 100% sure that I understand the request so I will ask for some clarification here.

1. I believe that when Jeff and I talked about the request he said to add it to the `guaccollect files` collector only, which is complete.
2. With `guacone collect files` I started to add it but I don't think it actually pushes the file to any blob store (I may be incorrect). So nothing to do here? (Perhaps the non-file collectors do add the file to the blob store bucket?)
3. I think that leaves deps_dev, gcs, git, github, oci, and s3 collectors. I believe you are asking me to provide the same command-line option to each of these collectors, and if the option is set, replace the node origin values with the blob store address instead of the original address?

Reply:
> With guacone collect files I started to add it but I don't think it actually pushes the file to any blob store (I may be incorrect). So nothing to do here? (Perhaps the non-file collectors do add the file to the blob store bucket?)


yes, correct `guacone` does not push to blob store. Sorry about that. 

> I think that leaves deps_dev, gcs, git, github, oci, and s3 collectors. I believe you are asking me to provide the same command-line option to each of these collectors, and if the option is set, replace the node origin values with the blob store address instead of the original address?

yes.

Reply:
We could do the other in another PR to close this one out.

Reply:
Let's do it in another PR. I will work on that shortly.